content/en/docs/tasks/traffic-management/ingress/ingress-control/index.md

```diff
@@ -95,40 +94,38 @@ $ export SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingress
 $ export TCP_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="tcp")].nodePort}')
 {{< /text >}}
 
-Setting the ingress IP depends on the cluster provider:
+_GKE:_
 
-1.  _GKE:_
-
-    {{< text bash >}}
-    $ export INGRESS_HOST=worker-node-address
-    {{< /text >}}
+{{< text bash >}}
+$ export INGRESS_HOST=workerNodeAddress
```

Comment:
why is this one camelcase, while all the rest of the instructions use dash separator for placeholders?

Reply:
I just copied from Getting Started for consistency between pages.

Reply:
Hmm, maybe it should be copied from here to getting started instead?

Reply:
Updated to dash operator on both pages for consistency.